README.md

```diff
@@ -132,6 +132,17 @@ To enable Google Anaytics, add the following lines to your Jekyll site:
 
 Google Analytics will only appear in production, i.e., `JEKYLL_ENV=production`
 
+### Link to specific files in header
+
+Add this to only link to the about and the linklists page in the default
+navigation header:
+
+```yaml
+header_pages:
+  - about.md
+  - linklists.md
+```
+
```

Comment:
I agree with @adamvoss, it's best to share the reason why one may want to use this option.